ssi-dids/src/lib.rs

```diff
@@ -948,7 +948,7 @@ impl FromStr for PrimaryDIDURL {
     type Err = Error;
     fn from_str(didurl: &str) -> Result<Self, Self::Err> {
         // Allow non-DID URL for testing lds-ed25519-2020-issuer0
-        #[cfg(test)]
+        #[cfg(not(feature = "testing-lds-ed25519-2020-issuer0"))]
```

Comment:
The feature `testing-lds-ed25519-2020-issuer0` doesn't exist -- but considering no test failed after introducing this invalid check, I think we can simply remove the gate altogether